src/api/handlers/deviceprofile.py

```diff
@@ -94,7 +96,7 @@ def log_device(self, request):
     if err:
       return err
 
-    self.__get_client_meta(request, args)
+    # self.__get_client_meta(request, args)
```

Comment:
remove all commented out code if not needed.

If you wish to leave any in there, leave a comment on why you prefer to keep the deadcode

Reply:
I had meant to get round to that. I must have missed a few of those. I'll go back and check.